### PR TITLE
Adding readme for hbase replication

### PR DESCRIPTION
**Pull request by vermas2012**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-bigtable-hbase/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

### CODE REVIEW

File: hbase-migration-tools/bigtable-hbase-replication/README.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+5. Immediately disable the CBT replication peer, this allows WAL logs to
+   accumulate on HDFS. On HBase shell execute:  `disable_peer '2'`
+6. Check the replicated tables by executing `list_replicated_tables` and enable
+   table level replication by executing `enable_table_replication "table_name"`
+7. Copy the existing data
+   using [offline migration tooling](https://cloud.google.com/architecture/hadoop/hadoop-gcp-migration-data-hbase-to-bigtable)
+8. After all the data is copied (and verified), enable CBT replication peer. On
+   HBase shell execute `enable_peer '2'`
+9. Eventually, replication will catch up and the 2 databases will be in sync. On
+   HBase shell execute `status 'replciation'` to check status of replication for
```
Comment (jhambleton) [inline hbase-migration-tools/bigtable-hbase-replication/README.md:73]: typo replication

File: hbase-migration-tools/bigtable-hbase-replication/README.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+       Cloud Bigtable Instance ID
+    </description>
+</property>
+```
+
+Please refer
+to [HBaseToCloudBigtableReplicationConfiguration](bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/configuration/HBaseToCloudBigtableReplicationConfiguration.java)
+for other properties that can be set. Please set other properties for
+authenticating to Cloud Bigtable based on
+your preferred [authentication](https://cloud.google.com/bigtable/docs/authentication)
```
Comment (jhambleton) [inline hbase-migration-tools/bigtable-hbase-replication/README.md:117]: is there a more prescriptive page for configuring authentication? For on-prem deployments, it is common for services to authenticate using service accounts, which we may require setting configurations such as `google.bigtable.auth.json.keyfile` in the hbase-site.xml

File: hbase-migration-tools/bigtable-hbase-replication/README.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+HBase 1.x clusters, please use bigtable-hbase-1.x-replication.jar, for HBase 2.x
+versions please use bigtable-hbase-2.x-replication.jar. Following are the steps
+to configure HBase to Cloud Bigtable replication:
+
+1. Install Cloud Bigtable replication library in HBase servers (both master and
+   region servers).
+    1. Download the library from maven on all the master and region server
+       nodes.
+    2. Add the library to the HBase region server class path. In DataProc, this
+       can be achieved by copying the library to /usr/lib/hbase/lib/
```
Comment (jhambleton) [inline hbase-migration-tools/bigtable-hbase-replication/README.md:132]: suggest removing dataproc to not confuse readers such as: 

Add the library to the HBase region server class path (i.e. `/usr/lib/hbase/lib/`)

File: hbase-migration-tools/bigtable-hbase-replication/README.md [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+2. Add CBT configs to hbase-site.xml
+3. Configure Cloud Bigtable authentication as
+   shown [here](https://cloud.google.com/bigtable/docs/authentication).
+4. Restart the HBase master, this allows the master to load the replication jar
+   and be aware of the classes in it.
```
Comment (jhambleton) [inline hbase-migration-tools/bigtable-hbase-replication/README.md:137]: can we update this step to match step #5, but for the master? they should be restarted in a rolling fashion.

File: hbase-migration-tools/bigtable-hbase-replication/README.md [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+|DeleteFamilyBeforeTimestamp|Not modified|Converts it to DeleteFamily if timestamp within a configurable threshold.|
+|DeleteFamily|Converts to DeleteFamilyBeforeTimestamp with timestamp=now|See DeleteFamilyBeforeTimestamp|
+|DeleteRow|Converts to DeleteFamilyBeforeTimestamp with timestamp=now for all families|See DeleteFamilyBeforeTimestamp|
+
+The goal of this replication library is to allow migration from HBase to CBT.
+Since CBT will not support these mutations after the users migrate to CBT, they
+are recommended to come up with alternative ways to handle these incompatible
+APIs and not issue them while replication is on.
+
+Another special case is mutations with reverse timestamps. HBase uses a `long`
```
Comment (jhambleton) [inline hbase-migration-tools/bigtable-hbase-replication/README.md:201]: link reverse timestamps to public docs https://hbase.apache.org/book.html#reverse.timestamp

**Review by jhambleton — CHANGES_REQUESTED** [review]
few minor comments, but overall this looks good.

File: hbase-migration-tools/bigtable-hbase-replication/README.md (hunk 2 above)
Comment (vermas2012) [reply hbase-migration-tools/bigtable-hbase-replication/README.md:117]: Maybe @igorbernstein2 or @kolea2  will know?

File: hbase-migration-tools/bigtable-hbase-replication/README.md (hunk 4 above)
Comment (vermas2012) [reply hbase-migration-tools/bigtable-hbase-replication/README.md:137]: Do on prem deployment usually have more than 1 master?

**Pull request closed without merging**